scripts/README.md

```diff
@@ -0,0 +1,68 @@
+
+# AutoDBA Scripts
+
+This directory contains a set of scripts for managing the AutoDBA project. Below is a summary of the provided scripts and their usage instructions.
+
+## Scripts
+
+1. **release.sh**: This script is used to build and release AutoDBA in different formats, such as `.tar.gz`, `.deb`, or `.rpm` packages.
+
+### Usage:
+```
+./release.sh [TARGET=all]
+```
+This will build the AutoDBA binaries for multiple architectures, prepare the web UI, include Prometheus exporters, and generate release packages in the `release_output` directory.
+
+The optional argument to this script can specify the target. The possible choices are:
+ - `all` (default)
+ - `tar.gz`
+ - `rpm`
+ - `deb`
+ - `source`
+
+2. **install.sh**: This script installs the AutoDBA project and its dependencies, including Prometheus. It supports both system-wide and user-specific installations.
+
+### Usage:
+```
+./install.sh [--system] [--install-dir <DIRECTORY>] [--package <PACKAGE_FILE>] [--config <CONFIG_FILE>]
+```
+- `--system`: Install system-wide under `/usr/local/autodba`. This flag also installs autodba as a service.
```

Comment:
Do we care about supporting user and system installation? The main reason I can think of is to run multiple AutoDBAs on the same machine, but I think you should be using some sort of container system (Docker or K8s) if you are going in that direction.

Reply:
The main difference is requiring `sudo` access if you want to install system-wide.

Reply:
Running multiple AutoDBAs on the same machine should also be possible with a single installation and different config files. We should also parameterize the port numbers in the config file for that. It's better to handle it separately.